boltdesignsystem/bolt
# 🌈 DS-465 colored chips

## Jira

https://pegadigitalit.atlassian.net/browse/DS-465

## Summary

Adds `color` and `border-radius` props to the Chip component.

## Details

1. `color` prop created. Status and a few select brand colors are added as options.
2. `border-radius` prop created. Now chips can be fully rounded or little rounded or not rounded at all.
3. Docs have been rewritten.

## How to test

Run the branch locally and check all the new Chip docs. Make sure all props work as expected.

## Release notes

### Visual Changes

- Chips have a slightly different style on hover. Text is darker and underlined, background is lighter. [See demo](http://localhost:3000/pattern-lab/?p=components-chip-url-variations) for reference.
> Note: update demo URL once this is released.
